plugins/RocketsPlugin/RocketsPlugin.cpp

```diff
@@ -47,36 +47,47 @@
 
 namespace
 {
-const std::string ENDPOINT_API_VERSION = "v1/";
+// REST PUT & GET, JSONRPC set-* notification, JSONRPC get-* request
 const std::string ENDPOINT_ANIMATION_PARAMS = "animation-parameters";
 const std::string ENDPOINT_APP_PARAMS = "application-parameters";
 const std::string ENDPOINT_CAMERA = "camera";
-const std::string ENDPOINT_FRAME_BUFFERS = "frame-buffers";
 const std::string ENDPOINT_GEOMETRY_PARAMS = "geometry-parameters";
-const std::string ENDPOINT_IMAGE_JPEG = "image-jpeg";
 const std::string ENDPOINT_TRANSFER_FUNCTION = "transfer-function";
 const std::string ENDPOINT_RENDERING_PARAMS = "rendering-parameters";
 const std::string ENDPOINT_SCENE = "scene";
 const std::string ENDPOINT_SCENE_PARAMS = "scene-parameters";
-const std::string ENDPOINT_SIMULATION_HISTOGRAM = "simulation-histogram";
-const std::string ENDPOINT_STATISTICS = "statistics";
 const std::string ENDPOINT_STREAM = "stream";
-const std::string ENDPOINT_STREAM_TO = "stream-to";
+const std::string ENDPOINT_VOLUME_PARAMS = "volume-parameters";
+
+// REST GET, JSONRPC get-* request
+const std::string ENDPOINT_STATISTICS = "statistics";
```

Comment:
So `'statistics'` and `'version'` can now be fetched using a jsonrpc request with `{method: 'get-statistics'}`/`{method: 'get-version'}`?

How about when statistics is broadcasted from the server? Is it broadcasted as `{method: 'get-statistics'}` or `{method: 'statistics'}`?

Reply:
Yes, they can be requested as you described.

The broadcast haven't changed, so it's ```statistics``` still